assets/training/distillation/components/pipeline/spec.yaml

```diff
@@ -1,6 +1,6 @@
 $schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
 name: oss_distillation_pipeline
-version: 0.0.7
+version: 0.0.8
```

Comment:
![image](https://github.com/user-attachments/assets/72d97d43-65b7-494b-bede-06a6540d99a4)


oss distillation is at version 5 in prod. Please set the version to 6

Reply:
Done.